[2025-02-02T12:36:32.544+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-02T12:36:32.561+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: housekeeping_job.execute_job manual__2025-02-02T19:30:06+07:00 [queued]>
[2025-02-02T12:36:32.570+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: housekeeping_job.execute_job manual__2025-02-02T19:30:06+07:00 [queued]>
[2025-02-02T12:36:32.571+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 3
[2025-02-02T12:36:32.586+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): execute_job> on 2025-02-02 12:30:06+00:00
[2025-02-02T12:36:32.595+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=583) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-02T12:36:32.594+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'housekeeping_job', 'execute_job', 'manual__2025-02-02T19:30:06+07:00', '--job-id', '318', '--raw', '--subdir', 'DAGS_FOLDER/housekeep.py', '--cfg-path', '/tmp/tmpsec9g4bl']
[2025-02-02T12:36:32.596+0000] {standard_task_runner.py:72} INFO - Started process 584 to run task
[2025-02-02T12:36:32.596+0000] {standard_task_runner.py:105} INFO - Job 318: Subtask execute_job
[2025-02-02T12:36:32.641+0000] {task_command.py:467} INFO - Running <TaskInstance: housekeeping_job.execute_job manual__2025-02-02T19:30:06+07:00 [running]> on host f03ab8549fa5
[2025-02-02T12:36:32.717+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='housekeeping_job' AIRFLOW_CTX_TASK_ID='execute_job' AIRFLOW_CTX_EXECUTION_DATE='2025-02-02T12:30:06+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-02T19:30:06+07:00'
[2025-02-02T12:36:32.719+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-02T12:36:39.665+0000] {logging_mixin.py:190} INFO - Successfully executed here after py ./jobs/first_DAG_jobs.py with batchdate: 2024-10-28
[2025-02-02T12:36:39.666+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-02T12:36:39.675+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-02T12:36:39.676+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=housekeeping_job, task_id=execute_job, run_id=manual__2025-02-02T19:30:06+07:00, execution_date=20250202T123006, start_date=20250202T123632, end_date=20250202T123639
[2025-02-02T12:36:39.706+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-02T12:36:39.720+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-02T12:36:39.722+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
